hostplumber/pkg/utils/ip/ip.go

```diff
@@ -0,0 +1,100 @@
+package ip
+
+import (
+	"github.com/vishvananda/netlink"
+)
+
+func GetIpv4Cidr(ifName string) (*[]string, error) {
```

Comment:
Minor nit: abbreviations are uppercased by convention in Go. 

```suggestion
func GetIPv4CIDR(ifName string) (*[]string, error) {
```

(holds for most of the functions in this file)

Reply:
Why do you return a pointer to the slice here instead of just the slice?